lib/modules/easterEgg.js

```diff
@@ -23,11 +21,10 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
+	});
 	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
 };
 
 export function konamiActive() {
-	return konami && konami.almostThere;
+	return konami;
```

Comment:
This is really the crux of the issue: this property is supposed to indicate if the user is 90-100% done with the Konami code sequence and RES shouldn't follow through with keyboard navigation. However,  It looks like  this version of the Konami library doesn't expose that directly. How about for this method, checking if konami exists and if Konami.input mostly matches konami.code?